pytest_splunk_addon/standard_lib/addon_parser/__init__.py

```diff
@@ -49,25 +45,15 @@ class AddonParser(object):
 
     def __init__(self, splunk_app_path):
         self.splunk_app_path = splunk_app_path
-        LOGGER.info(
-            f"Initializing the splunk_appinspect.App from path={splunk_app_path}"
-        )
-        self._app = None
         self._props_parser = None
         self._tags_parser = None
         self._eventtype_parser = None
         self._savedsearch_parser = None
 
-    @property
-    def app(self):
-        if not self._app:
-            self._app = App(self.splunk_app_path, python_analyzer_enable=False)
-        return self._app
-
     @property
     def props_parser(self):
```

Comment:
I'm curious if we could use lru_cache from https://docs.python.org/3.10/library/functools.html (yes, there are much nicer options in the documentation, but for python 3.8+) for this kind of constructions.

In this specific case, code could be more or less like below

```
@property
@lru_cache
def props_parser(self):
    return PropsParser(self.splunk_app_path)
```

Reply:
I'd say that we can introduce this change after we do a release and test it for couple of add-ons.